### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Jagannath-Padhy**
- fixed false error 
- added missing reason codes
- price pattern in schema

### CODE REVIEW

File: schema/Retail/Cancel/on_cancel_rto.ts [hunk 1]
```diff
@@ -491,7 +492,8 @@ export const cancel_RTO_Schema = {
                     },
                     value: {
                       type: 'string',
-                    },
+                      pattern: '^[0-9]+(.[0-9]{1,2})?$',
+                      errorMessage: 'Price value should be a number in string with upto decimal places'                    },
```
Comment (sandeepshahi) [inline schema/Retail/Cancel/on_cancel_rto.ts:496]: upto 2* decimal places

File: schema/Retail/Cancel/on_cancel_rto.ts [hunk 2]
```diff
@@ -526,7 +528,8 @@ export const cancel_RTO_Schema = {
                           },
                           value: {
                             type: 'string',
-                          },
+                            pattern: '^[0-9]+(.[0-9]{1,2})?$',
+                            errorMessage: 'Price value should be a number in string with upto decimal places'                          },
```
Comment (sandeepshahi) [inline schema/Retail/Cancel/on_cancel_rto.ts:532]: 2* decimal places

File: schema/Retail/Cancel/on_cancel_rto.ts [hunk 3]
```diff
@@ -541,7 +544,8 @@ export const cancel_RTO_Schema = {
                               },
                               value: {
                                 type: 'string',
-                              },
+                                pattern: '^[0-9]+(.[0-9]{1,2})?$',
+                                errorMessage: 'Price value should be a number in string with upto decimal places'                              },
```
Comment (sandeepshahi) [inline schema/Retail/Cancel/on_cancel_rto.ts:548]: pls rectify in all the occurrences

Comment (Jagannath-Padhy) [reply schema/Retail/Cancel/on_cancel_rto.ts]: done

**Pull request merged**